Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6396,42 +6396,58 @@ def com_google_fonts_check_metadata_category_hint(family_metadata):
     id = "com.google.fonts/check/colorfont_tables",
     rationale = """
         Colr v0 fonts are widely supported in most browsers so they do not require
-        an SVG color table. However, Colr v1 is only well supported in Chrome so
-        we need to add an SVG table to these fonts.
+        an SVG color table. However, Colr v1 is currently only well supported in
+        Chrome so we need to add an SVG table to these fonts, except in the case of
```

Comment:
I wouldn't say only well supported in Chrome.. Also Microsoft Edge (also based on Chromium so shares same Skia-based implementation of COLRv1) and FireFox (has its own independent implementation) support COLRv1 well.

Reply:
Thanks for the clarification, will fix.